src/GapicClientTrait.php

```diff
@@ -75,6 +75,7 @@ trait GapicClientTrait
         Call::CLIENT_STREAMING_CALL => 'startClientStreamingCall',
         Call::SERVER_STREAMING_CALL => 'startServerStreamingCall',
     ];
+    private static $defaultAudience = 'https://' . self::SERVICE_ADDRESS . '/';
```

Comment:
This throws exceptions in testing so a `getDefaultAudience` method may be better.